### PR TITLE
Port getEnvOrDie{,inProduction} from TypeScript to Python

### PR DESCRIPTION
**Pull request by lpsinger**
These are Python ports of utility functions in this TypeScript file: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/lib/env.server.ts

### CODE REVIEW

**Comment by swyatt7** [comment]
The code looks good to me. Where will this will be implemented?

**Review by swyatt7 — APPROVED** [review]

**Comment by lpsinger** [comment]
Unfortunately I don't think that `NODE_ENV` is defined for the Python Lambdas, so I need to change this to use `ARC_ENV`.

**Comment by lpsinger** [comment]
> Where will this will be implemented?

I don't understand the question.

File: python/env.py
Comment (jak574) [inline python/env.py:1]: Where do you expect these new functions to be called from? Right now this pull won't result in code that gets run.

Comment (lpsinger) [reply python/env.py:1]: Here, for one: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/python/across_api/base/tle.py#L196-L197

Comment (jak574) [reply python/env.py:1]: So the intention is to replace `os.environ.get` with these calls?

Comment (lpsinger) [reply python/env.py:1]: Yes.

**Review by swyatt7 — APPROVED** [review]

**Comment by swyatt7** [comment]
Looks good

**Pull request merged**